src/plugins/discover/public/application/components/sidebar/string_progress_bar.tsx

```diff
@@ -17,35 +17,24 @@
  * under the License.
  */
 import React from 'react';
-import { EuiFlexGroup, EuiFlexItem, EuiProgress, EuiText, EuiToolTip } from '@elastic/eui';
+import { EuiProgress, EuiToolTip } from '@elastic/eui';
 
 interface Props {
   percent: number;
   count: number;
+  value: string;
 }
 
-export function StringFieldProgressBar(props: Props) {
+export function StringFieldProgressBar({ value, percent, count }: Props) {
+  const tooltipContent = `${value} ${count}`;
   return (
     <EuiToolTip
```

Comment:
I think it's a bit odd that the Tooltip with the value name and the count (vs percentage) is only wrapped around the progress element and not the whole thing. Is it possible to move this tooltip to wrap the whole field value, percentage and progress (being sure not to wrap it around the filter icons)?

Reply:
Haha 15 seconds later than @kertal's same comment 😆